rdfproxy/utils/sparql_utils.py

```diff
@@ -20,21 +20,21 @@
 """)
 
 
-def replace_query_select_clause(query: str, repl: str) -> str:
+def replace_query_select_clause(query: str, repl: str):
```

Comment:
why does it not return a string anymore?

Reply:
oversight. I added the return annotation.